test/connectors/Sub09ConnectorSpec.scala

```diff
@@ -111,6 +111,15 @@ class Sub09ConnectorSpec extends SpecBase {
       }
     }
 
+    "return xi eori information from the api when pbeaddress is empty" in new Setup {
+      when(mockHttp.GET[Option[MdgSub09XiEoriInformationResponse]](any[URL], any[Seq[(String, String)]])(any(), any(), any()))
+        .thenReturn(Future.successful(Some(mdgXiEoriInformationResponse(Sub09Response.noXiEoriAddressInformation(testEori)))))
+
+      running(app) {
```

Comment:
Better to replace (from line 118-120) as below

      running(app) {
        service.getXiEoriInformation(testEori).map {
          xiInfo => xiInfo mustBe Some(xiEoriInformationWithNoAddress)
        }
      }

Reply:
Updated as per the comment